pdns/dnsdist-lua-vars.cc

```diff
@@ -69,6 +69,20 @@ void setupLuaVars()
       {"Additional",3 }
     });
 
+  g_lua.writeVariable("EDNSOptionCode", std::unordered_map<string,int>{
+      {"NSID",          3 },
```

Comment:
Just a nit, but perhaps this would be better:
```
{ "NSID",          EDNSOptionCode::NSID },
...
```

Reply:
yes!

Reply:
pushed